core/src/test/java/io/questdb/griffin/O3CommitLagTest.java

```diff
@@ -319,7 +319,9 @@ public void testVarColumnPageBoundariesAppend() throws Exception {
     public void testVarColumnPageBoundariesAppendRndPageSize() throws Exception {
         int rndPagesMultiplier = new Rnd(Os.currentTimeMicros(), Os.currentTimeNanos()).nextInt(129);
         int multiplier = Numbers.ceilPow2(rndPagesMultiplier);
+
         dataAppendPageSize = (int) Files.PAGE_SIZE * multiplier;
+        dataAppendPageSize = 262144;
```

Comment:
This line seems to overwrite the value stored by the previous one. Is that on purpose?

Reply:
debris, removing it

Reply:
changed in https://github.com/questdb/questdb/pull/1778/commits/3b8609ba3442fa0a590cd19693726b231d02cbff